scripts/check-form.js

```diff
@@ -19,16 +19,18 @@ if (!key) {
 
 const sheetData = `https://sheets.googleapis.com/v4/spreadsheets/${sheetID}/values/Sheet1!A2:A?key=${key}`;
 
-const [,, slug, branch, all] = process.argv;
+const [,, slug, all, ref = 'HEAD'] = process.argv;
```

Comment:
This looks like you can't pass `ref` without also passing `--all`?

What if we used the actual CLI argument parser instead:

```js
const { values, positionals } = require('util').parseArgs({
  options: { all: { type: 'boolean', default: false } },
  allowPositionals: true,
});
if (positionals.length !== 2) throw 'args required: slug, ref';
const [slug, ref] = positionals;
const { all } = values;
```

Reply:
yeah, that's a fair point, i'll switch to use that.